### ISSUE TITLE
Error in Android Device

### ISSUE THREAD

**Issue by danielmao** [capacitor-community/google-maps#97]
**Describe the bug**
The app crashes in Android when it goes to background and is opened multiple times. I found this error y Android Native

E/AndroidRuntime: FATAL EXCEPTION: main
    Process:  PID: 25927
    java.lang.RuntimeException: Unable to stop activity {com.......}: java.lang.IllegalStateException
        at android.app.ActivityThread.callActivityOnStop(ActivityThread.java:4995)
        at android.app.ActivityThread.performStopActivityInner(ActivityThread.java:4960)
        at android.app.ActivityThread.handleStopActivity(ActivityThread.java:5040)
        at android.app.servertransaction.StopActivityItem.execute(StopActivityItem.java:41)
        at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:176)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2216)
        at android.os.Handler.dispatchMessage(Handler.java:107)
        at android.os.Looper.loop(Looper.java:237)
        at android.app.ActivityThread.main(ActivityThread.java:7948)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1075)
     Caused by: java.lang.IllegalStateException
        at com.google.android.libraries.maps.ij.zzae.zzb(Preconditions.java:29)
        at com.google.android.libraries.maps.ke.zzae.zze(MapRendererPhoenix.java:253)
        at com.google.android.libraries.maps.ka.zzba.zzC(GoogleMapImpl.java:310)
        at com.google.android.libraries.maps.ka.zzcw.zzh(MapViewDelegateImpl.java:16)
        at com.google.android.libraries.maps.MapView$zza.onStop(MapView.java:18)
        at com.google.android.gms.dynamic.DeferredLifecycleHelper.onStop(com.google.android.gms:play-services-base@@17.3.0:63)
        at com.google.android.libraries.maps.MapView.onStop(MapView.java:39)
        at com.hemangkumar.capacitorgooglemaps.CapacitorGoogleMaps.handleOnStop(CapacitorGoogleMaps.java:90)
        at com.getcapacitor.Bridge.onStop(Bridge.java:1092)
        at com.getcapacitor.BridgeActivity.onStop(BridgeActivity.java:149)
        at android.app.Instrumentation.callActivityOnStop(Instrumentation.java:1474)
        at android.app.Activity.performStop(Activity.java:8187)
        at android.app.ActivityThread.callActivityOnStop(ActivityThread.java:4987)
        at android.app.ActivityThread.performStopActivityInner(ActivityThread.java:4960) 
        at android.app.ActivityThread.handleStopActivity(ActivityThread.java:5040) 
        at android.app.servertransaction.StopActivityItem.execute(StopActivityItem.java:41) 
        at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:176) 
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2216) 
        at android.os.Handler.dispatchMessage(Handler.java:107) 
        at android.os.Looper.loop(Looper.java:237) 
        at android.app.ActivityThread.main(ActivityThread.java:7948) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1075) 

**To Reproduce**
1. Put Background 
2. Open App 
3. Do 1, 2 multiple times

**Comment by danielmao** [comment]
Someone can reproduce this error? is this plugin constantly updated?

**Comment by tafelnl** [comment]
We've just released a (pretty stable) beta for version 2 of this plugin under the name [`@capacitor-community/google-maps`](https://www.npmjs.com/package/@capacitor-community/google-maps).

It features a more complete and consistent API. Also the map can finally be rendered behind the webview in this new version.

Extensive documentation for v2 can be found here: https://capacitor-community.github.io/google-maps/

Since version 1 lacked a lot of (basic) features, did not have good coding practices, had a poor API and it is not recommended to use it anymore, we've dropped support for it. Therefore I will close this issue. We believe version 2 offers a better Developer Experience and is a much better fit for anyones use case.
